### DISCUSSION TITLE
keep导出gpx数据失败

### DISCUSSION THREAD

**Discussion by ymyguang** [yihong0618/running_page#229] [General]
报错原因：无法访问数据库

**Comment by yihong0618** [comment]
能提供下截图么？
理论上不会出现这个问题

**Comment by ben-29** [comment]
这个问题我遇到过，脚本应该从根目录运行，从 `scripts` 目录运行会有点问题
你试试从项目根目录运行 `python3(python) scripts/keep_sync.py` 看还会不会有这个问题 @ymyguang
